packages/parameterized_test/example/parameterized_test_example.dart

```diff
@@ -1,55 +1,92 @@
-import 'package:parameterized_source/parameterized_source.dart';
 import 'package:parameterized_test/parameterized_test.dart';
 import 'package:test/test.dart';
 
 void main() {
   parameterizedTest(
-    'Amount of letters',
-    ParameterizedSource.csv([
-      'kiwi, 4',
-      'apple, 5',
-      'banana, 6',
-    ]),
-        (List<dynamic> values) {
-      final String input = values[0];
-      final expected = values[1];
-
-      final actual = input.length;
+    'list list dynamic wrong type cast',
+    [
+      ['kiwi', 14],
+      ['apple', 15],
+      ['banana', 16],
+    ],
+    p2((String word, bool length) {
+      expect(word.length, length);
+    }),
+  );
 
-      expect(actual, expected);
+  parameterizedTest(
+    'list list dynamic list to long for selected test',
+    [
+      ['kiwi', 4, 1],
+      ['apple', 5, 1],
+      ['banana', 6, 1].withTestOptions(skip: 'not now'),
+    ],
+    p2((String word, int length) {
```

Comment:
It also feels weird that this now just plainly ignores the third item in the list.

Reply:
Can this test even be generated? This *should* throw an error because you provide more options then the test method accepts right?